src/containers/TargetsMetrics.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import MetricsList from '../presentational/metric/MetricsList'
+import {calculateMetrics} from '../actions/metricsActions'
+
+class TargetsMetrics extends Component {
```

Comment:
this component to display the progress bars for each macro after calculating the totals based on the day entries.